src/components/tag-input/tag-input.component.html

```diff
@@ -39,7 +39,7 @@
 </ol>
 
 <!-- Insert the custom icon if provided -->
-<div class="ux-tag-icons" *ngIf="icon || _showClearButton">
+<div class="ux-tag-icons" *ngIf="icon || _showClearButton"(click)="toggle(); $event.stopPropagation()">
```

Comment:
Can you add a space between the " and the (click

Reply:
Sure thing, just firing this up to see if it builds without issues